hermes_memory_provider/__init__.py

```
@@ -3371,6 +3375,47 @@ def on_memory_write(self, action: str, target: str, content: str) -> None:
     # MNEMOSYNE_SHUTDOWN_DRAIN_TIMEOUT.
     SHUTDOWN_DRAIN_TIMEOUT_SECONDS = _parse_env_float("MNEMOSYNE_SHUTDOWN_DRAIN_TIMEOUT", 2)
 
+    # L3 persona file injection. Default OFF -- opt in via
+    # MNEMOSYNE_PERSONA_ENABLED=true. When OFF, no file IO happens.
+    PERSONA_ENABLED = _parse_env_bool("MNEMOSYNE_PERSONA_ENABLED", False)
```

Ruff (0.15.20)

[warning] Boolean positional value in function call

(FBT003)